Magento_Theme/web/js/custom-post-form.js

```diff
@@ -0,0 +1,35 @@
+define(['jquery', 'mage/mage'], function ($) {
+  $.widget('vaimo.customPostForm', {
+    options: {
+      actionUrl: 'https://app.exampleproject.test/testform/index/submit',
```

Comment:
actionUrl has to be only `testform/index/submit`. the rest of the URL has to be generated by the `mage/url` utility